src/SearchIndex/util.ts

```diff
@@ -462,7 +462,6 @@ async function getManifestsFromS3(bucketName: string, prefix: string): Promise<M
   const manifests = [];
   for (const bucketEntry of result.Contents || []) {
     if (bucketEntry.Size === 0) {
```

Comment:
can we actually validate its not a "directory file" ie doesnt end in a trailing slash, and log it if so?